httpx/utils.py

```diff
@@ -102,3 +103,25 @@ def get_netrc_login(host: str) -> typing.Optional[typing.Tuple[str, str, str]]:
 
     netrc_info = netrc.netrc(str(netrc_path))
     return netrc_info.authenticators(host)  # type: ignore
+
+
+def parse_header_links(value: str) -> typing.List[typing.Dict[str, typing.Any]]:
```

Comment:
The typing is a little loose here, can we tighten this up to remove the `typing.Any`?
Also can we add a docstring to this function along with an example?

Reply:
changed the typing to str and added a docstring